api/registrations/views.py

```diff
@@ -380,6 +381,8 @@ class RegistrationContributorsList(BaseContributorList, RegistrationMixin, UserM
     """
     view_category = 'registrations'
     view_name = 'registration-contributors'
+
+    pagination_class = NodeContributorPagination
```

Comment:
This change looks like it affects the registration endpoint more generally, but the test you added only is for institution registrations. Can you add a comparable one to `test_registration_list` please?